### ISSUE TITLE
Feature Request: conditional dropout

### ISSUE THREAD

**Issue by forestsource** [kohya-ss/sd-scripts#45]
Hi!
Thank you for your great works.

It would be nice if you could support for conditional dropout.
https://github.com/victorchall/EveryDream2trainer/blob/c816e2577379060cd1015b6a1bca5a109049ab6a/doc/ATWEAKING.md#conditional-dropout
https://github.com/victorchall/EveryDream2trainer/blob/98f9a7302d3325354e69bfa87f3851e98c4735e9/data/every_dream.py#L145-L156

**Comment by lolxdmainkaisemaanlu** [comment]
I think Stable Diffusion 1.5 was finetuned with 10% conditional dropout. That lets it work across a wide range of Classifier Free Guidance (CFG) values.

I think this implementation would be very helpful for training and give better results.

**Comment by HoiMinJeong** [comment]
ドロップアウトによって結果が非常に大きく異なる方です。 設定した確率に応じてキャプションとタグを空で学習させる単純なコードなので、早く適用されるといいと思います。
